packages/runtime/container-runtime/src/summaryCollection.ts

```diff
@@ -209,11 +214,17 @@ export class SummaryCollection {
     private pendingAckTimerTimeoutCallback: (() => void) | undefined;
     private lastAck: IAckedSummary | undefined;
 
-    public get latestAck() { return this.lastAck; }
+    public get latestAck(): IAckedSummary | undefined { return this.lastAck; }
+
+    public get opsSinceLastAck() {
+        return this.deltaManager.lastSequenceNumber -
+            (this.lastAck?.summaryAck.sequenceNumber ?? this.deltaManager.initialSequenceNumber);
+    }
 
     public constructor(
         private readonly deltaManager: IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>,
         private readonly logger: ITelemetryLogger,
+        private readonly opActions: Partial<SummaryCollectionOpActions>,
```

Comment:
```suggestion
        private readonly opActions: SummaryCollectionOpActions,
```
The type itself is already `Partial`, so this is redundant. Should change this or the definition depending on need.